stable/external-dns/templates/clusterrole.yaml

```diff
@@ -18,4 +18,12 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
Would it be possible to do this more like this:
```
{{- range .Values.sources }}
{{- if eq . "istio-gateway" }}
  - apiGroups:
      - networking.istio.io
    resources:
      - gateways
    verbs:
      - get
      - list
      - watch
{{- end }}
{{- end }}
```

I think we shouldn't be adding access to that unless it is needed.

Reply:
@sudermanjr Thanks I missed that, I added the condition to the PR.